### PR TITLE
Blogapp: Feature/setup

### PR DESCRIPTION
**Pull request by mateo951**
- Added rails setup and DB for postgres
- Added linters
- Modified gitignore file
- Added controllers and routes
- Added readme file description

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>Posts#index</h1>
+<p>Find me in app/views/posts/index.html.erb</p>
```
Comment (Dipeshtwis) [inline app/views/posts/index.html.erb:2]: - [ ] There should be corresponding view files for each of the controller actions. Currently, show view file is missing for post. [Hint: `app/views/posts/show.html.erb `]

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Users#index</h1>
+<p>Find me in app/views/users/index.html.erb</p>
```
Comment (Dipeshtwis) [inline app/views/users/index.html.erb:2]: - [ ] There should be corresponding view files for each of the controller actions. Currently, the show view file is missing for the user. [Hint: `app/views/users/show.html.erb `]

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @mateo951, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (mateo951) [reply app/views/posts/index.html.erb:2]: Done ✔

File: app/views/users/index.html.erb (hunk 2 above)
Comment (mateo951) [reply app/views/users/index.html.erb:2]: Done ✔

**Comment by Davidosky007** [comment]
## Hi @mateo951 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
done-gif



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**